docs/api/sm-api.md

```diff
@@ -166,8 +193,8 @@ At the current time, filtering supports the following comparisons:
 | lteq     | Less than or equal to |
 | gteq     | Greater than or equal to |
 | not_eq   | Not equal |
+| match    | Inexact match (case-insensitive substring match) |
```

Comment:
it should also support eq, which is distinct from match (ie. Exact Match) of the entire string that was provided vs a partial match.